### PR TITLE
Development merge to Master

### PR DESCRIPTION
**Pull request by ssro**
Various improvements

### CODE REVIEW

**Comment by T-Dnzt** [comment]
:+1:

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Jason Wilder
+Copyright (c) 2014 Jason Wilder (c) 2015 Sebastian Sasu
```
Comment (T-Dnzt) [inline LICENSE]: What year is it now?

**Comment by whereismybuild** [comment]
https://blog.martinfjordvald.com/2011/04/optimizing-nginx-for-high-traffic-loads/

Not needed for our current traffic patterns, but here's an example of a config that can handle 'all the connections!' (about 76k per second)

.. and if we start getting 76k connections to the backend.. other things will start falling over.. like the DB ;)

<snip>
worker_processes  4;  # 2 * Number of CPUs

events {
    worker_connections  19000;  # It's the key to high performance - have a lot of connections available
}

worker_rlimit_nofile    20000;  # Each connection needs a filehandle (or 2 if you are proxying)

# Total amount of users you can serve = worker_processes * worker_connections
</snip>

File: LICENSE (hunk 1 above)
Comment (ssro) [inline LICENSE]: I started working on it in 2015 T-Dnzt

**Comment by ssro** [comment]
I've seen that page @whereismybuild . The only thing I've changed are the worker processes, because the servers have 2 cpu's. The rest is pretty much unchanged in the config file. Indeed the nginx won't be a problem for high traffic loads.

**Pull request merged**